code/game/gamemodes/cult/cult_items.dm

```diff
@@ -127,3 +127,50 @@
 
 /obj/item/clothing/suit/space/cult/cultify()
 	return
+
+/obj/item/weapon/melee/cultknife
+	name = "ritual knife"
+	desc = "A wicked looking curvy dagger with sharp edges. It seems to be stained with blood."
+	icon_state = "ritualknife"
+	item_state = "cultknife"
+	w_class = 2
+	force = 12
+	throwforce = 10
+	edge = 1
+	sharp = 1
+	hitsound = 'sound/weapons/bladeslice.ogg'
+
+	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")

```

Comment:
Are these (and most of the above vars) just copied from an existing knife item? If so, you could just make it a child type of that.